pvHelpers/user.py

```diff
@@ -6,11 +6,12 @@
 
 
 class OrganizationInfo(object):
-    def __init__(self, organiztion_id, organiztion_name, department_name, role):
-        self.org_id = organiztion_id
-        self.org_name = organiztion_name
+    def __init__(self, organization_id, organization_name, department_name, role, show_mua_prompt = True):
```

Comment:
I understand why having a default value here makes it easier not to change so much codes in daemon.

However, it can introduce weird inconsistencies since we sometimes read org_info from the local user store.
And from the changes in core, the value will be defaulted to True.

So @davey-h how difficult is it to change all `OrganizationInfo` in core, so we don't have default values?
@samantehrani I let you comment on this one since we have a lot of discussion around default value?

Reply:
@hu13 there doesn't look to me many constructors for it used in the app (think 2 or 3) but then there are 10+ cases of it in tests. I don't mind updating the other 1 or 2 to make sure it matches the logic, and maybe just saying `True` in tests?